src/System-Support/SystemDictionary.class.st

```diff
@@ -275,6 +278,13 @@ SystemDictionary >> hasClassOrTraitNamed: aString [
 	^ false
 ]
 
+{ #category : #'dictionary access' }
+SystemDictionary >> lookupVar: name [
+	"Return a var with this name.  Return nil if none found"
+	name isString ifFalse: [ ^nil ].
```

Comment:
I know it was there before but I would really remove this check. It should be an explicit error if type protection is really required.

Reply:
yes, I added it already in the todo, but I wanted to do it independently of this change